PyPiDescription.rst

```diff
@@ -59,8 +89,8 @@ Testing
 
 After installation, try out the MMS Quickstart for `Serving a
 Model <https://github.com/awslabs/mxnet-model-server/blob/master/README.md#serve-a-model>`__
-and `Exporting a
-Model <https://github.com/awslabs/mxnet-model-server/blob/master/README.md#export-a-model>`__.
+and `Creating a
+Model Archiver<https://github.com/awslabs/mxnet-model-server/blob/master/README.md#model-archive>`__.
```

Comment:
Should probably be called create-a-model-archive ..

Reply:
done